source/index.md

```diff
@@ -97,7 +115,13 @@ IIIF Specifications are created and published following the [IIIF Editorial Proc
 [prezi-jp]: http://www.asahi-net.or.jp/~ax2s-kmtn/ref/iiif/apipresentation2.1.html
 [search-jp]: http://www.asahi-net.or.jp/~ax2s-kmtn/ref/iiif/searchapi1.0.html
 [auth-jp]: http://www.asahi-net.or.jp/~ax2s-kmtn/ref/iiif/apiauthentication1.0.html
-
+[image3-chinese]: https://www.yuque.com/iiifchina/df4qfk/fwybkl
+[prezi3-chinese]: https://www.yuque.com/iiifchina/df4qfk/gpf6od
+[auth1-chinese]: https://www.yuque.com/iiifchina/df4qfk/vkxifz
+[contentsearch1-chinese]: https://www.yuque.com/iiifchina/df4qfk/ygbnck
+[discovery1-chinese]: https://www.yuque.com/iiifchina/df4qfk/hdb26g
+[contentstate-09-chinese]: https://www.yuque.com/iiifchina/df4qfk/wysy7i
+[navplace-chinese]: https://iiif.io/api/extension/navplace/
```

Comment:
The `navplace-chinese` link points to the English language version

Reply:
Great catch, I will have to get the right link from Meg on Monday